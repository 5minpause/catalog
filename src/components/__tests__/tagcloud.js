import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import TagCloud from "../tagcloud"

beforeEach(() => {
  useStaticQuery.mockImplementationOnce(() => (
    {
        "allMarkdownRemark": {
          "group": [
            {
              "fieldValue": "ruby",
              "totalCount": 2
            },
          ],
        },
    }
  )
)})

describe("TagCloud", () =>
  it("renders correctly", () => {
    const tree = renderer.create(<TagCloud />).toJSON()
    expect(tree).toMatchSnapshot()
  }))