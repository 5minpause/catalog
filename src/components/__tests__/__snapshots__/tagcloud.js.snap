// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagCloud renders correctly 1`] = `
<div
  style={
    Object {
      "padding": 12,
    }
  }
>
  <ul
    style={
      Object {
        "marginBottom": 96,
        "paddingLeft": 0,
      }
    }
  >
    <li
      style={
        Object {
          "backgroundColor": "rgba(0, 0, 240, 0.3)",
          "borderRadius": 6,
          "display": "inline",
          "listStyleType": "none",
          "margin": 12,
          "padding": 6,
        }
      }
    >
      <a
        href="/tags/ruby/"
        style={
          Object {
            "color": "#000",
            "textDecoration": "none",
          }
        }
      >
        ruby
         (
        2
        )
      </a>
    </li>
  </ul>
</div>
`;
